Functions.Templates/Templates/CosmosDBTrigger-CSharp-5.x/.template.config/template.json

```diff
@@ -0,0 +1,78 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Azure Function", "Trigger", "Cosmos DB"],
+  "name": "CosmosDBTrigger",
+  "identity": "Azure.Function.CSharp.CosmosDBTrigger.2.x",
+  "groupIdentity": "Azure.Function.CosmosDBTrigger",
+  "shortName": "cosmos",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "sourceName": "CosmosDBTriggerCSharp",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    },
+    "ConnectionStringSetting": {
+      "description": "The name of the App Setting containing the connection string to the service that contains the collection to be monitored.",
+      "type": "parameter",
+      "defaultValue": "",
+      "replaces": "ConnectionValue"
+    },
+    "DatabaseName": {
+      "description": "This is the name of the database that includes the collection to be monitored.",
+      "type": "parameter",
+      "defaultValue": "databaseName",
+      "replaces": "DatabaseValue"
+    },
+    "CollectionName": {
+      "description": "This is the name of the collection to be monitored.",
+      "type": "parameter",
+      "defaultValue": "collectionName",
+      "replaces": "CollectionValue"
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
+    }
+  },
+  "primaryOutputs": [
+    {
+      "path": "CosmosDBTriggerCSharp.cs"
+    }
+  ],
+  "defaultName": "CosmosDBTriggerCSharp",
+  "postActions": [
+    {
+      "Description": "Adding Reference to Microsoft.Azure.WebJobs.Extensions.CosmosDB Nuget package",
+      "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+      "ContinueOnError": "true",
+      "ManualInstructions": [],
+      "args": {
+        "referenceType": "package",
+        "reference": "Microsoft.Azure.WebJobs.Extensions.CosmosDB",
+        "version": "4.0.0",
```

Comment:
Can we use the latest version 4.3.0?

Reply:
Yes, this is a pretty old PR at this point, and the latest package at the time was 4.0.0. Updated to `4.3.0`